climada/entity/exposures/crop_production.py

```diff
@@ -105,42 +92,45 @@
 FAO_FILE = "FAOSTAT_data_producer_prices.csv"
 FAO_FILE2 = "FAOSTAT_data_production_quantity.csv"
 
+YEARS_FAO = np.array([2000, 2018])
+"""Default years from FAO used"""
+
 # default output directory: climada_python/data/ISIMIP_crop/Output/Exposure
 # by default the hist_mean files created by climada_python/hazard/crop_potential are saved in
 # climada_python/data/ISIMIP_crop/Output/hist_mean/
 HIST_MEAN_PATH = os.path.join(DATA_DIR, 'ISIMIP_crop', 'Output', 'Hist_mean')
 OUTPUT_DIR = os.path.join(DATA_DIR, 'ISIMIP_crop', 'Output')
 
 
-class CropyieldIsimip(Exposures):
+class CropProduction(Exposures):
     """Defines agriculture exposures from ISIMIP input data and
     FAO crop price data"""
 
     _metadata = Exposures._metadata + ['crop']
 
     @property
     def _constructor(self):
-        return CropyieldIsimip
+        return CropProduction
 
     def set_from_single_run(self, input_dir=INPUT_DIR, filename=None, hist_mean=HIST_MEAN_PATH,
-                            bbox=BBOX, yearrange=(YEARCHUNKS[SCENARIO[1]])['yearrange'],
-                            cl_model=None, scenario=SCENARIO[1], crop=CROP[0], irr=IRR[0],
+                            bbox=BBOX, yearrange=(YEARCHUNKS['histsoc'])['yearrange'],
+                            cl_model=None, scenario='histsoc', crop=None, irr=None,
                             unit='USD', fn_str_var=FN_STR_VAR):
 
         """Wrapper to fill exposure from nc_dis file from ISIMIP
         Parameters:
             input_dir (string): path to input data directory
-            filename (string): name of the file to use
-            historic mean (array): historic mean yield per centroid
+            filename (string): name of the landuse data file to use,
+                e.g. "histsoc_landuse-15crops_annual_1861_2005.nc""
+            hist_mean (str or array): historic mean crop yield per centroid (or path)
             bbox (list of four floats): bounding box:
                 [lon min, lat min, lon max, lat max]
             yearrange (int tuple): year range for exposure set
                 f.i. (1990, 2010)
             scenario (string): climate change and socio economic scenario
-                f.i. 'histsoc' or 'rcp60soc'
-            cl_model (string): abbrev. climate model (only when landuse data
-            is future projection)
-                f.i. 'gfdl-esm2m' etc.
+                f.i. '1860soc', 'histsoc', '2005soc', 'rcp26soc','rcp60soc','2100rcp26soc'
+            cl_model (string): abbrev. climate model (only for future projections of lu data)
+                f.i. 'gfdl-esm2m', 'hadgem2-es', 'ipsl-cm5a-lr','miroc5'
```

Comment:
The change from global variables to this docstring is very good. Could one be more precise here and not use 'f.i.' only? The examples given here refer to one particular dataset right?